packages/flutter_test/lib/src/controller.dart

```diff
@@ -217,6 +316,309 @@ class SemanticsController {
 
     return false;
   }
+
+  /// Performs the given [SemanticsAction] on the given [SemanticsNode]
```

Comment:
document the arguments?

Reply:
Forgot that I had to add more arguments to this after the first writing... Good catch! Updated!